cmd/agentctl/main.go

```diff
@@ -474,6 +475,7 @@ config that may be used with this agent.`,
 	cmd.Flags().StringVarP(&stackID, "stack", "u", "", "stack ID to get a config for")
 	cmd.Flags().StringVarP(&apiKey, "api-key", "p", "", "API key to authenticate against Grafana Cloud's API with")
 	cmd.Flags().StringVarP(&apiURL, "api-url", "e", "", "Grafana Cloud's API url")
+	cmd.Flags().StringVar(&platforms, "platforms", "", "Platforms/OSes to retrieve configuration for")
```

Comment:
Pluralization feels a little weird here. Is it valid to provide multiple platforms? If so, how does the user provide more than one? Comma-delimited?

Reply:
We can add it later if necessary. For now, I think changing the help text of the flag to note that the platforms are comma-delimited would be good 👍